src/main/java/emu/grasscutter/utils/ConfigContainer.java

```diff
@@ -136,6 +136,7 @@ public static class Game {
         public int bindPort = 22102;
         /* This is the port used in the default region. */
         public int accessPort = 0;
+        public boolean enableScriptInBigWorld = false;
```

Comment:
What's the point of making this controllable? I can't imagine situation where you wouldn't want this to be enabled, and we already depend on Lua for other stuff so...

Reply:
it's experimental and there are many Lua functions not implemented currently in the big world and it will produce lots of errors. 
So I make it disable default so people can open it if they really need it.